vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/CredentialIssuer.kt

```diff
@@ -94,9 +110,16 @@ class CredentialIssuer(
         clientNonceService = clientNonceService,
     ),
 ) {
+    private val supportedSigningAlgorithms = cryptoAlgorithms
+        .mapNotNull { it.toJwsAlgorithm().getOrNull()?.identifier }.toSet()
+
     private val supportedCredentialConfigurations = credentialSchemes
-        .flatMap { it.toSupportedCredentialFormat(cryptoAlgorithms).entries }
-        .associate { proofValidator.associateWithProofTypes(it) }
+        .flatMap { it.toSupportedCredentialFormat().entries }
+        .associate {
+            it.key to it.value
```

Comment:
`.associate { (key, value) -> key to value`?

Reply:
don't see the increased readability here